.github/workflows/build-and-deploy.yml

```diff
@@ -3,9 +3,12 @@ on:
   push:
     branches:
       - master
+  schedule:
+    # Run at 6 AM Eastern (11:00 UTC)
+    - cron: '0 11 * * *'
```

Comment:
hmm wonder if it would be better to just run this monday to friday. I think it should be fine as it is, but just in case of anything we will be around on the weekdays.